xpreports/build_windows.go

```diff
@@ -3,10 +3,11 @@ package xpreports
 import (
 	"strconv"
 
+	"github.com/1dustindavis/gosal/version"
```

Comment:
hey @1dustindavis thanks for this, sorry i got to it so late!

qq, should the package path be in your namespace or the airbnb namespace?